source/includes/documentation/_vue_js_elements.md

```diff
@@ -3486,3 +3486,48 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.39 Sheets
+
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/sheets/)
+
+- __Java__: _com.epam.jdi.light.vuetify.elements.composite.Sheet.java_
+
+```java
+    //@FindBy(css = "#RoundedSheet")
+    @UI("#RoundedSheet")
+    public static Sheet roundedSheet;
+    
+    @Test
+    public void roundedSheetTest() {
+        roundedSheet.show();
+        roundedSheet.is().displayed()
+            .is().rounded().is().shaped()
+            .has().borderRadius(4).and().elevation(4)
+            .has().color("rgba(253, 216, 53, 1)")
+            .has().height(150).and().has().width(150)
+            .has().gradient();
+    }
+```
+
+The Sheet component is a transformable "piece of paper" that provides a basic foundation for Vuetify features.
+For example, properties such as rounded and shaped modify the border-radius property while elevation increases/decreases box-shadow.
+
+![Sheets examples](../../images/vuetify/sheets.png)
+
+__Vuetify v2.6.14__ code example:
+```html
+<div class="d-flex v-sheet theme--light elevation-4 rounded yellow lighten-3" style="height: 150px; width: 150px;">
+  <div class="mt-auto align-center justify-center d-flex px-2 v-sheet theme--dark" style="height: 50px; background-color: rgba(0, 0, 0, 0.36); border-color: rgba(0, 0, 0, 0.36);">
+    Rounded XL, Elevation 4
+  </div>
+</div>
+```
+
+| Method | Description | Return Type | 
+|:---| :--- |:---|
+**borderRadius()** | Returns sheet border radius in px | int
+**color()** | Returns sheet color (the same as backgroundColour()) | [](overridden method)
+
+Sheet also have basic JDI elements methods for Color, Elevation, Measurements, Theme and others
+
+For examples of usage see: [JDI Vuetify Sheets tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SheetsTests.java).
```

Comment:
link does not work, get '404'

Reply:
It should work after https://github.com/jdi-testing/jdi-light/pull/4783 merge.